packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/Timeline/BasicTimeline.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Timeline from '@cerner/terra-graphs/lib/components/Timeline/Timeline';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import getTimelineConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Timeline/timelineConfigObject';
+import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Timeline/defaultDataset.js';
+
+/*
+Please refer to the documentation below to see the graphConfig and data objects
+*/
+const graphConfig = utils.deepClone(getTimelineConfig('#simpleLinegraph'));
```

Comment:
The id should be something related to the timeline graph, like **SimpleTimelineGraph**.

Reply:
updated here: b1898007ef72fda908cd5275ee447282f5906a56